av/src/Video.cc

```diff
@@ -194,9 +194,9 @@ bool Video::Load(const std::string &_filename)
     return false;
   }
 
-#if LIBAVCODEC_VERSION_MAJOR < 60
   // Inform the codec that we can handle truncated bitstreams -- i.e.,
   // bitstreams where frame boundaries can fall in the middle of packets
+#if LIBAVCODEC_VERSION_MAJOR < 60
```

Comment:
it will be easier to resolve conflicts during future forward ports with the comments outside the macro block